### PR TITLE
Added csv2 and csv3 serializer. Modified csv-serializer

### LINKED ISSUE #119: Add norwegian CSV-serializers in PCAxis.Common Nuget
**Include 2 new serializers, and minor change to the existing CsvFileSerializer.vb**
   WrapTextWithQuote was not used in a corner case in the existing.
  DataCellNotes is now removed, so that e.g. 3.14e becomes 3.14 so that summing in Excel works.

**The new formats**
A format where each cell gets its own row:
CSV2: Pivoted model with all variables in stub 
CSV3: Pivoted model with all variables in stub, shorter description (code instead of text)

### PR DESCRIPTION
**Pull request by PerIngeVaaje**
fixes statisticssweden/PxWeb#119

CSV2: Pivoted model with all variables in stub
CSV3: Pivoted model with all variables in stub, shorter description (code instead of text)

CSV:Remove data cell note from data cell

### CODE REVIEW

File: PCAxis.Common/Serializers/CsvFileSerializer.vb [hunk 1]
```diff
@@ -262,22 +310,45 @@ Namespace PCAxis.Paxiom
...
+                sb.Append(Model.Meta.Stub(stubIndex).Values(valueIndex).Text)
+            Else
+                sb.Append(Model.Meta.Stub(stubIndex).Values(valueIndex).Code)
+            End If
             If Me.WrapTextWithQuote Then sb.Append(ControlChars.Quote)
 
             Return sb.ToString
         End Function
 
+        Protected Function GetPivotedModelAllStub(ByVal oldModel As PXModel) As Paxiom.PXModel
```
Comment (likp) [inline PCAxis.Common/Serializers/CsvFileSerializer.vb]: Feels like this is misplaced, maybe this function should be extracted to its own utility class.

Comment (PerIngeVaaje) [reply PCAxis.Common/Serializers/CsvFileSerializer.vb]: Maybe in Misc-folder?

Comment (likp) [reply PCAxis.Common/Serializers/CsvFileSerializer.vb]: Sounds like the right place.

**Review by likp — APPROVED** [review]
Looks great now

**Pull request merged**